packages/atlas/src/components/_inputs/TextArea/TextArea.styles.ts

```diff
@@ -1,30 +1,39 @@
 import styled from '@emotion/styled'
 
 import { Text } from '@/components/Text'
-import resizeIcon from '@/components/_icons/svgs/action-resize.svg'
 import { cVar, sizes } from '@/styles'
 
-export const StyledTextArea = styled.textarea`
+import { InputSize, getBaseInputStyles, getInputPadding } from '../inputs.utils'
+
+export const TextAreaWrapper = styled.div`
+  display: inline-block;
+  width: 100%;
+`
+
+export const StyledTextArea = styled.textarea<{ inputSize: InputSize; error?: boolean }>`
   width: 100%;
   resize: vertical;
+  overflow-y: auto;
+  overflow-x: hidden;
+
+  ${getBaseInputStyles}
+
+  ${getInputPadding}
+
+  ::-webkit-scrollbar-corner {
+    background: ${cVar('colorCoreNeutral800Lighten')};
+  }
 
   ::-webkit-resizer {
-    background: url(${resizeIcon}) no-repeat;
+    /* action-resize.svg with updated color */
+    background-image: url("data:image/svg+xml,%3Csvg width='16' height='16' viewBox='0 0 16 16' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill-rule='evenodd' clip-rule='evenodd' d='M14 7.00003C14 6.59557 13.7564 6.23093 13.3827 6.07615C13.009 5.92137 12.5789 6.00692 12.2929 6.29292L7.29292 11.2929C7.00692 11.5789 6.92137 12.009 7.07615 12.3827C7.23093 12.7564 7.59557 13 8.00003 13H13C13.5523 13 14 12.5523 14 12V7.00003Z' fill='%237B8A95'/%3E%3C/svg%3E");
```

Comment:
Any specific reason to embed the svg here directly?

Reply:
Haha, that's a long story! How about we discuss it over a beer in two weeks? :D (just kidding, I'll let @drillprop do the talking, but let me just add it's some next-level trickery he's doing here to make it work as intended)

Reply:
- I tried to use SVG in the component, but this didn't work well when the scrollbar was visible.
- Importing SVG and placing it in URL as we did before(`background: url(${resizeIcon}) no-repeat;`) didn't work either, because we needed to set different `fill` color.  We would have to add SVG to some folder, manually change its color and then import it. 

That's why this is converted SVG by URL-encoder. I didn't find a better solution to do this.